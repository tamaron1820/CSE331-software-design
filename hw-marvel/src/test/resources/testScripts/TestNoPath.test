# Test if two nodes does not have any path

# To run this test, the marvel dataset must exist at
#   hw-marvel/src/main/resources/data/soccerPlayer.csv

LoadGraph soccer soccerPlayer.csv

FindPath soccer Messi Haland