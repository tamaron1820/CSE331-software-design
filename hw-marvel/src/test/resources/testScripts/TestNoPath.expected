# Test if two nodes does not have any path

# To run this test, the marvel dataset must exist at
#   hw-marvel/src/main/resources/data/soccerPlayer.csv

loaded graph soccer

path from Messi to Haland:
no path found